gemrb/GUIScripts/iwd/CharGen.py

```diff
@@ -1524,17 +1524,21 @@ def SkillsPress():
 	LUSkillsSelection.SkillsNullify (MyChar)
 
 	if SkillsState == 0:
-		GemRB.SetVar ("HatedRace", 0)
-		if IsThief!="*":
-			SkillsSelect()
-		elif DruidSpell!="*":
+		if DruidSpell!="*":
```

Comment:
Aren't you still giving druids stealth this way? 

Besides the reported spacing issues also keep in mind the space before an opening paren.

Reply:
Yes, it's the same condition as it was previously. Thief if was moved down below. If mod adds RacialEnemy for thief, RacialEnemySelect will be triggered, which in turn will trigger later SkillsSelect.

Reply:
> Besides the reported spacing issues also keep in mind the space before an opening paren.

Should all method/function calls have space before opening parenthesis? Some do, some not.

Reply:
Yes, it's one of the oldest and wierdest style decisions in the project.

I forgot this turned into an enhancement request as well, not just about fixing the initial bug.